src/v/raft/probe.cc

```diff
@@ -156,6 +162,14 @@ void probe::setup_metrics(const model::ntp& ntp) {
           [this] { return _full_heartbeat_requests; },
           sm::description("Number of full heartbeats sent by the leader"),
           labels),
+        sm::make_histogram(
```

Comment:
I remember there have been warnings about adding new per-ntp histograms because they put undue stress on the metrics-collecting infrastructure (even when aggregated across partitions). As this is more of a performance diagnostics metric, maybe we can get by with a single shard-local histogram without per-ntp granularity?

Reply:
ok, removing this for now as discussed offline. Perhaps would be nice to have a mode that a user can enable on the fly (temporarily) to collect more granular metrics.